spec/views/application_templates/edit_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+include RSpecHtmlMatchers
+
+describe 'application_templates/edit.haml' do
+  before :each do
+    @template = create :application_template
+    @top_question = create :question, application_template: @template
+    @middle_question = create :question, application_template: @template
+    @bottom_question = create :question, application_template: @template
+  end
+  it 'includes a link to view application' do
+    render
+    action_path = application_template_path @template
+    expect(rendered).to have_tag 'a', with: { href: action_path }
+  end
+  it 'contains a form with questions' do
```

Comment:
Those inputs are part of that form, so your test should look for that. In other words, not just that there's an input called prompt anywhere on the page, but that there's one specifically in that form.